ui/app/templates/components/license-banners.hbs

```diff
@@ -1,24 +1,23 @@
 {{#if this.licenseExpired}}
   <div class="license-banner-wrapper" data-test-license-banner data-test-license-banner-expired>
     <AlertBanner
-      @bannerType="license"
       @type="danger"
       @title="License expired"
       @message="Your Vault license expired on {{date-format
         @expiry
         'MMM d, yyyy'
       }}. Add a new license to your configuration and restart Vault."
-      @marginless={{true}}
+      class="message-marginless"
     >
       <DocLink @path="/vault/tutorials/enterprise/hashicorp-enterprise-license">
+        <Icon @name="learn-link" />
```

Comment:
added to keep consistent with the new styling. this is the only other alert-message with a yielded DocLink.